site/docs/v1/tech/shared/_deleting-your-account.adoc

```diff
@@ -0,0 +1,20 @@
+To delete your account, take the following steps, in this order:
+
+1. delete all deployments
+2. switch your account to the community plan
+3. file a support ticket
+
+[NOTE.note]
+====
+FusionAuth require that customers delete their own deployments using the account portal.
+
+FusionAuth is **not able** to cancel accounts with deployments that could potentially contain critical user data.
+====
+
+To remove your deployments, please https://account.fusionauth.io/account/deployment/[log into your account and navigate to your deployments]. Once you see your deployment, you can click the [uielement]#Action# button and select [uielement]#Delete#. After a deployment is destroyed, you won't be charged for it.
```

Comment:
After a deployment is destroyed, you won't be charged for it going forward -- any previous usage outside of a trial will be billed according to the stated billing terms.